dynamodb/src/main/kotlin/com/playkids/onboarding/dynamodb/DynamoDBItemDAO.kt

```diff
@@ -56,25 +56,25 @@ class DynamoDBItemDAO(config: Config, private val dynamoDbClient: DynamoDbAsyncC
             CATEGORY to category.toAttributeValue(),
             ID to id.toAttributeValue(),
             IMAGE to image.toAttributeValue(),
-            COIN_PRICE to coinPrice.toAttributeValue(),
-            GEM_PRICE to gemPrice.toAttributeValue()
+            CURRENCY to currency.toString().toUpperCase().toAttributeValue(),
```

Comment:
[nit] you could use name property
```suggestion
currency.name.toAttributeValue()
```